newsroom/agenda/views.py

```diff
@@ -161,7 +165,7 @@ async def get_view_data() -> Dict:
         "locators": get_vocabulary("locators"),
         "ui_config": await ui_config_service.get_section_config("agenda"),
         "groups": get_groups(get_app_config("AGENDA_GROUPS", []), company_dict),
-        "has_agenda_featured_items": get_resource_service("agenda_featured").find_one(req=None) is not None,
+        "has_agenda_featured_items": await FeaturedService().find_one(req=None) is not None,
```

Comment:
We now have a get all function on Async services, would be good to use that instead